src/main/java/com/github/hcsp/Home.java

```diff
@@ -11,5 +11,8 @@ public static Cat newCat(String name) {
         // Create a new Cat instance, assign the name to its "name" member variable,
         // and return it here
         // 在这里创建一个新的Cat实例，将其成员变量"name"设为参数所指定的名字，并返回之
+        Cat cat = new Cat();
+        cat.name = "name";
```

Comment:
```suggestion
        cat.name = name;
```

"name"是一个字符串常量，不是"参数所指定的名字".